gammapy/data/event_list.py

```diff
@@ -438,15 +441,15 @@ def plot_energy(self, ax=None, **kwargs):
 
         Parameters
         ----------
-        ax : `~matplotlib.axes.Axes` or None
-            Axes
-        **kwargs : dict
-            Keyword arguments passed to `~matplotlib.pyplot.hist`
+        ax : `~matplotlib.axes.Axes`, optional
+            Matplotlib axes. Default is None
```

Comment:
I think it is fine to leave it as 'Axes', otherwise it is repeating a bit the line above

Reply:
There are a number of instances of this type

Reply:
Not doing this as discuss